projects/aca-content/src/lib/components/search/search-results/search-results.component.ts

```diff
@@ -338,4 +348,16 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
     const updatedUserQuery = formatSearchTerm(this.searchedWord, this.searchConfig['app:fields']);
     this.queryBuilder.userQuery = updatedUserQuery;
   }
+
+  private shouldExecuteQuery(navigationStartEvent: NavigationStart | null, params: Params): boolean {
+    if (navigationStartEvent === null) {
+      return true;
+    } else if (navigationStartEvent.navigationTrigger === 'imperative') {
+      return false;
+    } else if (navigationStartEvent.navigationTrigger === 'popstate' || navigationStartEvent.navigationTrigger === 'hashchange') {
+      return true;
```

Comment:
the same return is in 354th line (first if) so to avoid repeating return please combine both ifs into single if. So to do that you can just add conditions for navigationTrigger to first if using ||. Then true will be returned when navigationStartEnd is null or navigationTrigger is popstate or navigationTrigger is hashchange